05_syntax_lab/07.py

```diff
@@ -1,7 +1,17 @@
-""" Write a program that selects a random number
-and asks the user to guess it.
-After each guess print a hint "too large" or "too small" to the user.
-Bonus: To make things interesting, the program should cheat once in a white
-"""
-
-
+""" create random number between 0-100, and let the user guess it with our help"""
+from random import randint
+x = randint(1,100)
+y = 0
+i = 0
+while y != x:
+    print "please insert number:"
+    y = raw_input()

```

Comment:
Also I think it'll be nicer to convert to int here instead of in every use, i.e. `y = int(raw_input('Please insert a number: '))`